.github/workflows/pr.yaml

```diff
@@ -26,12 +30,33 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  authorize:
+    # The 'external' environment is configured with the maintainers team as required reviewers.
+    # All the subsequent jobs in this workflow 'need' this job, which will require manual approval for PRs coming from external forks.
+    # see list of approvers in OWNERS file
+    environment:
+      ${{ github.event.pull_request.head.repo.full_name == github.repository && 'internal' || 'external' }}
+    runs-on: ubuntu-latest
+    outputs:
+      is_internal: ${{ steps.check.outputs.is_internal }}
+    steps:
+      - name: Check if internal PR
+        id: check
+        run: |
+          if [[ "${{ github.event.pull_request.head.repo.full_name }}" == "${{ github.repository }}" ]]; then
+            echo "is_internal=true" >> $GITHUB_OUTPUT
+            echo "✓ Internal PR - will run with PR code"
+          else
+            echo "is_internal=false" >> $GITHUB_OUTPUT
```

Comment:
where are these variables being used in the script? below this line I only see `authorize` being called.